src/getHumanAge.test.js

```diff
@@ -7,4 +7,21 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('animals` human age should be 0 if their own age < 15', () => {
+    expect(getHumanAge(0, 14)).toEqual([0, 0]);
+  });
+
+  test('animals` human age should be 1 if their own age > 14 and < 24', () => {
+    expect(getHumanAge(15, 23)).toEqual([1, 1]);
+  });
+
+  test(`cat's age should add 1 every 4 years after 23 years old
+    dog's age should add 1 every 5 years after 23 years old`, () => {
+    expect(getHumanAge(24, 24)).toEqual([2, 2]);
+    expect(getHumanAge(27, 28)).toEqual([2, 2]);
+    expect(getHumanAge(31, 33)).toEqual([3, 3]);
+    expect(getHumanAge(59, 59)).toEqual([10, 9]);
+    expect(getHumanAge(47, 88)).toEqual([7, 14]);
+  });
```

Comment:
The logic described in the test case for ages after 23 might be incorrect or misunderstood. Ensure that the `getHumanAge` function correctly implements the logic where a cat's age adds 1 every 4 years and a dog's age adds 1 every 5 years after 23 years old. The expectations in the test cases should reflect this logic accurately.